packages/fury-adapter-apiary-blueprint-parser/lib/adapter.js

```diff
@@ -9,10 +9,10 @@ const mediaTypes = [
 
 const detect = source => mediaTypes.indexOf(deckardcain.identify(source)) !== -1;
 
-const parse = (options, done) => {
+function parse(options) {
   const parser = new Parser(options);
-  parser.parse(done);
-};
+  return Promise.resolve(parser.parse());
```

Comment:
Maybe we should also reject in case `parser.parse()` throws an exception? So we could use `.catch()` on `parse` invocations:

```js
parse(options).catch(console.error)
```